packages/dds/tree/src/edit-manager/editManager.ts

```diff
@@ -54,13 +59,6 @@ export class EditManager<TChangeset, TChangeFamily extends ChangeFamily<any, TCh
     }
 
     public addSequencedChange(newCommit: Commit<TChangeset>): Delta.Root {
-        if (this.trunk.length > 0) {
```

Comment:
We could potentially still assert that the sequence number is greater than the previous commit's sequence number, but I don't know how useful the assert would be.

Reply:
Agreed. Done.